database/seeder.js

```diff
@@ -199,7 +199,7 @@ const seed = async () => {
           potableWater: {
             available: false,
             types: 0,
-            description: null
+            description: ''
```

Comment:
How come you decided to change this from null to a string?

Reply:
This was for that "graceful error handling" that Leslie was talking about. I was noticing that my app would just die when it was seeing null in lieu of an empty string.... but I haven't figured it out fully yet.